hack/create-service-account-secrets.sh

```diff
@@ -5,7 +5,7 @@
 # - hiveadmission
 # - hive-controllers
 #
-cat <<EOF | kubectl apply -f -
+cat <<EOF | oc apply -f -
```

Comment:
-1

This script is explicitly for folks setting up hive on non-openshifts.

And most of the time, people have them (sym)linked to each other anyway.

```
efried@efried-thinkpadp16vgen1:~/go/src/github.com/openshift/hive$ ll -i ~/bin/oc ~/bin/kubectl
2567104 -rwxr-xr-x. 2 efried efried 160441224 Jul 18 16:42 /home/efried/bin/kubectl*
2567104 -rwxr-xr-x. 2 efried efried 160441224 Jul 18 16:42 /home/efried/bin/oc*
```